test/test_networking.py

```diff
@@ -1126,19 +1140,6 @@ def test_clean_header(self):
             assert 'Youtubedl-no-compression' not in rh.headers
             assert rh.headers.get('Accept-Encoding') == 'identity'
 
-    def test_build_handler_default_params(self):
-        with FakeYDL() as ydl:
-            rh = self.build_handler(ydl)
-            assert rh.headers.items() == std_headers.items()
-            assert rh.timeout == 20.0
-            assert rh.source_address is None
-            assert rh.verbose is False
-            assert rh.prefer_system_certs is False
-            assert rh.verify is True
-            assert rh.legacy_ssl_support is False
-            assert rh.client_cert is None
-            assert rh.cookiejar is ydl.cookiejar
```

Comment:
If we are removing this, then we need to add the following in test_build_handler_params:

- rh.cookiejar is ydl.cookiejar
- rh.headers includes std_headers